backend/btrixcloud/colls.py

```diff
@@ -537,6 +537,7 @@ async def list_collections(
         sort_direction: int = 1,
         name: Optional[str] = None,
         name_prefix: Optional[str] = None,
+        has_dedupe_index: Optional[bool] = None,
```

Comment:
@tw4l did a pass at adding collection dedupe filter, let me know if there's any changes to make, or feel free to push changes.

Reply:
Looks great! We could add a test for this but we'll be doing a pass on having backend test coverage before merging the feature branch to main so can add then.